src/DevLicenseCore.sol

```diff
@@ -184,6 +198,34 @@ contract DevLicenseCore is IDevLicenseDimo, AccessControl {
         emit SignerDisabled(tokenId, signer);
     }
 
+    /**
+     * @notice Internal function to set an alias to a token ID
+     * @param tokenId The unique identifier for the license token.
+     * @param licenseAlias The alias string to be set
+     */
+    function _safeSetLicenseAlias(uint256 tokenId, bytes32 licenseAlias) internal {
+        if (_aliasToTokenId[licenseAlias] != 0) {
```

Comment:
This is the same as the `licenseAlias` being an empty string, so it will make _aliasToTokenId[''] to be 0 and then it will set the `licenseAlias` normally. This might be the case when you want to "delete" you license alias